setup.py

```diff
@@ -9,6 +9,13 @@
 from setuptools import find_packages, setup
 
 PROJECT_PATH = os.path.dirname(os.path.abspath(__file__))
+_available_cuda_versions = [
+    "101",
+    "102",
+    "110",
+    "111",
+]  # TODO: align these with what's available in JAX before release
+_jax_version_constraints = ">= 0.2.11"
```

Comment:
nit: no space after `">=`

Does the latest jax release support `jax[cpu]` syntax? If not, I think we can wait until after the next numpyro release.

Reply:
The `[cpu]` and `[tpu]` targets for jax are consistently available since jax version 0.2.11. However, I just noted that prior to 0.2.13 the GPU targets are `[cuda-xxx]` and since 0.2.13 its `[cudaxxx]`, which is a bit annoying. It might make sense to wait with this until we the jax dependency is bumped to 0.2.13 or above.

Reply:
Please feel free to pump the minimum jax version to `0.2.13` and jaxlib to `0.1.65`. Currently, colab is using jax 0.2.13 and jaxlib 0.1.66 so its users won't have to reinstall (which potentially leads to conflicts) jax/jaxlib there.